content/news/core-monthly/2023/core-monthly-2023-11-01-2023-11-30.md

```diff
@@ -0,0 +1,420 @@
+
+Speaking of Live Updade, we didn't have it last month due to the fact that everyone was focused on the developer conference, but we are back this month! The next Live Update for the PrestaShop Project will happen on **Wednesday, December 20th, 2023** at [4 pm (Paris time)](https://time.is/1600_20_Dec_2023_in_Paris) on [YouTube](https://www.youtube.com/watch?v=7CQ2Sg-v4XQ).
+
+During the Live Update, we will go over the latest project updates and present two topics from the developer conference. We'll share more details soon.
+
+### PrestaShop 8.1.3
+
+The next patch version of PrestaShop 8.1.3 was originally scheduled to be released in December, but due to the fact that we are still working on this version we decided to postpone the release to January.
+
+What can you expect from 8.1.3? There are quite a few bugfixes, including some significant ones. One pull request fixes the [bug of missing stock change information for combinations](https://github.com/PrestaShop/PrestaShop/pull/34625) for modules. This could have caused a lot of problems because the Hook informing them of such a fact was not running correctly.
+
+Also [fixed a bug that caused JavaScript problems](https://github.com/PrestaShop/PrestaShop/pull/34600) and thus product management in the back office on the new product page.
+
+There's more so for a full list of changes you should definitely take a look [here](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+milestone%3A8.1.3+is%3Amerged).
+
+### PrestaShop 9
+
+Work on the new major version of PrestaShop is not slowing down what you can see by reviewing the list of code changes that have been merged into the core. As usual, the main release is a great opportunity to clean up some things and improve the existing ones. It's also the big new features you've been waiting for.
+
+In the changes that are being made you can see a great deal that is related to the migration of the back office to Symfony. The SymfonyLayout project still requires a few things but the whole project is slowly approaching completion. You can see [the details in EPIC available on our GitHub](https://github.com/PrestaShop/PrestaShop/issues/32875).
+
+The other big project that is part of PrestaShop 9 is, of course, the implementation of [ApiPlatform](https://api-platform.com/) to create a full-fledged API for the back office. It's a big project, but the whole thing is taking shape and as those who had the opportunity to be at the PrestaShop developers conference already know - the feature will be available in PrestaShop 9 and thanks to its modularity it will be possible for the whole community to work on making the API coverage better, so that back office events like adding products, categories, changing configurations or managing discounts, will be implemented as soon as possible. If you want to know more about the API, I invite you to the next [Live Update](https://www.youtube.com/watch?v=7CQ2Sg-v4XQ). where we will talk about it in more detail.
```

Comment:
```suggestion
The other big project that is part of PrestaShop 9 is, of course, the implementation of [ApiPlatform](https://api-platform.com/) to create a full-fledged API for the back office. It's a big project, but the whole thing is taking shape. As those who had the opportunity to be at the PrestaShop Developer Conference already know, the feature will be available in PrestaShop 9. Thanks to its modularity, it will be possible for the whole community to work on making a better API coverage, so back office events like adding products, categories, changing configurations or managing discounts, will be implemented as soon as possible. If you want to know more about the API, I invite you to the next [Live Update](https://www.youtube.com/watch?v=7CQ2Sg-v4XQ), where we will talk about it in more detail.
```